packages/contracts/contracts/AggregatorRegistry.sol

```diff
@@ -0,0 +1,26 @@
+pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
+
+/* External Imports */
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
+
+/* Internal Imports */
+import { Aggregator } from "./Aggregator.sol";
+
+/* Maintains the list of aggregators */
+contract AggregatorRegistry {
+  address[] public aggregators;
+
+  /* Add a new aggregator to this contract */
```

Comment:
I’d consider using this documentation format: https://github.com/plasma-group/pigi/blob/master/packages/contracts/contracts/Deposit.sol#L80-L84 — I think it’s called “NatSpec” ? https://solidity.readthedocs.io/en/develop/natspec-format.html & it’s what is commonly used to document solidity.

Reply:
Awesome, I will use this moving forward. Thanks Karl :)